### PR TITLE
xxhsum: add page

### PR DESCRIPTION
**Pull request by suyashbhawsar**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/xxhsum.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# xxhsum
+
+> Print or verify checksums using fast non-cryptographic algorithm xxHash.
+> More information: <https://github.com/Cyan4973/xxHash>.
+
+- Print checksums with algorithm selection:
+
+`xxhsum -H{{0 | 32 | 64 | 128}} path/to/file`
```
Comment (kbdharun) [inline pages/linux/xxhsum.md:8]:
```suggestion
`xxhsum -H{{0 | 32 | 64 | 128}} {{path/to/file}}`
```

File: pages/linux/xxhsum.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# xxhsum
+
+> Print or verify checksums using fast non-cryptographic algorithm xxHash.
+> More information: <https://github.com/Cyan4973/xxHash>.
+
+- Print checksums with algorithm selection:
+
+`xxhsum -H{{0 | 32 | 64 | 128}} {{path/to/file}}`
```
Comment (navarroaxel) [inline pages/linux/xxhsum.md:8]: This is the correct format
```suggestion
`xxhsum -H{{0|32|64|128}} {{path/to/file}}`
```

File: pages/linux/xxhsum.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# xxhsum
+
+> Print or verify checksums using fast non-cryptographic algorithm xxHash.
+> More information: <https://github.com/Cyan4973/xxHash>.
+
+- Print checksums with algorithm selection:
```
Comment (navarroaxel) [inline pages/linux/xxhsum.md:6]:
```suggestion
- Calculate the checksum for a file using a specific algorithm:
```

**Review by navarroaxel — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**